vllm/v1/executor/multiproc_executor.py

```diff
@@ -118,41 +131,60 @@ def _init_executor(self) -> None:
 
         # Initialize worker and set up message queues for SchedulerOutputs
         # and ModelRunnerOutputs
-        max_chunk_bytes = envs.VLLM_MQ_MAX_CHUNK_BYTES_MB * 1024 * 1024
-        self.rpc_broadcast_mq = MessageQueue(
-            self.world_size, self.world_size, max_chunk_bytes=max_chunk_bytes
-        )
-        scheduler_output_handle = self.rpc_broadcast_mq.export_handle()
+        self._init_request_rpc_mq()
```

Comment:
I think it would be a bit clearer to actually inline some of the logic here, like:
```python
            self.rpc_broadcast_mq = None
            self.scheduler_output_handle = None
            if self.parallel_config.distributed_node_rank_within_dp == 0:
                # Only create outbound broadcast queue if we are the leader node
                # within a dp rank, not if we are a headless multiproc executor.
                self.rpc_broadcast_mq = self._init_request_rpc_mq()
                scheduler_output_handle = self.rpc_broadcast_mq.export_handle()
```
and `scheduler_output_handle` doesn't need to be a field.